### PR TITLE
fix: I removed shopify styles to reuse other webapp classes

### PR DESCRIPTION
**Pull request by GusBaamonde**
#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

https://cdn.fromdoppler.com/doppler-ui-library/build671/integrations.html

layout-shopify

Related to: https://github.com/FromDoppler/doppler-webapp/pull/628

### CODE REVIEW

File: assets/scss/templates/_integrations.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   margin-top: 20px;
 
   @at-root {
-    &__block {
+    /* &__block {
```
Comment (andresmoschini) [inline assets/scss/templates/_integrations.scss]: What do you think about directly removing this code in place of commented it?

Comment (GusBaamonde) [reply assets/scss/templates/_integrations.scss]: Yes, at the moment I am doing tests to see it on the webapp, I promise to delete everything that is not used

**Comment by cbernat** [comment]
👍

**Comment by cbernat** [comment]
🚀

**Pull request merged**